bot.md

```diff
@@ -0,0 +1,85 @@
+
+_Doing this steps in order is more likely to result in success_
+
+### 0. Prerequisites
+
+- Install [Telegram Desktop](https://desktop.telegram.org/)
+- In the Telegram app, [Enable Web View Debug](https://core.telegram.org/bots/webapps#debug-mode-for-mini-apps)
+- Obtain a Telegram Bot [Token](https://core.telegram.org/bots/tutorial#obtain-your-bot-token)
+- Get a Pretix API token from a member of the 0xPARC PCD team.
+
+### 1. Set up .env
+
+- Run `cp apps/passport-server/.env.local.example apps/passport-server/.env`
+- Paste the Telegram Bot Token in `apps/passport-server/.env` for the `TELEGRAM_BOT_TOKEN` value
+- Paste the Pretix Token in `apps/passport-server/.env` for the `PRETIX_TOKEN` value
+- Confirm that the current value for `PRETIX_ORG_URL` is correct. (Most likely will be `https://pretix.eu/api/v1/organizers/pcd-0xparc`)
+
+### 2. Start the Passport Client and Server
+
+- `yarn`
+- `yarn build`
+- `yarn localdb:init && yarn localdb:up`
+- `yarn dev:bot`
+
+### 3. Sync the Dev Tickets
+
+- `cd apps/passport-server && yarn scratch new-dev-event`
+  - _Note: You can call `yarn scratch new-dev-event <eventId> <activeItemIds>` to make a custom event_
+- Copy the output from the `scratch` script (ex: `/link ProgCrypto (Internal Test)`)
+- You will paste the output in the following step.
+
+### 4. PCD Join / Auth flow in Telegram
+
+- Make a new private group chat in the Telegram app (ex: `test chat`).
+- Go to the chat with your bot (ex: https://t.me/zulearn_bot)
+- Go to `Info`, then click `Add to group or channel`. Add the bot to `test chat`
+- Now, in `test chat`, type `/link`. You should get a message confirming that the test event was linked to the TG chat.
+- You can double check by using a GUI like Postico to view the `telegram_bot_events` table.
+
+### 5. Putting it All Together
+
+- In the Telegram app, go to the chat with your bot and type `/start`
+- Click `Generate ZKP`
+- Follow the link to PCDPass within Telegram (http://localhost:3000) and hit `Prove` when your ticket appears
+- If this is your first time, you will need to make a new PCDPass account:
+
+  - Make a new PCDPass account with an email you **know** has a ticket for the event in question (`ivan@0xparc.org` should work).
+  - _Note: This account will only exist locally on your device_
+
+- You should be returned to the Telegram app and presented with the `Send ZKP` option.
+- Click `Send ZKP`, then you will be redirected to `test_chat`
+
+### 6. (optional) Hot Reloading for TG Development
+
+This is a one-time setup for serving https locally. You need it for Telegram, because Telegram hates http and localhost. At the end, you will be able to access 127.0.0.1 (aka localhost) from a fake domain, `dev.local` (this is important for https)
+
+1. Install mkcert - [https://github.com/FiloSottile/mkcert](https://github.com/FiloSottile/mkcert)
+2. `mkcert -install`
+3. From root of repo: `mkdir apps/certificates && cd apps/certificates && mkcert dev.local`
```

Comment:
nit: i prefer sticking to a convention of "descriptive text" in the number header and then code block below. code blocks are generally preferred for situations where we want devs to copy CLI commands

```suggestion
3. Now, run the follow commands from the repo root to generate a new certificate
```bash
mkdir apps/certificates && cd apps/certificates && mkcert dev.local
```

```